test/unit/openshift/cluster.test.ts

```diff
@@ -125,7 +240,28 @@ suite('Openshift/Cluster', () => {
                 expect(result).equals(`Successfully logged in to '${testUrl}'`);
             });
 
-            test('exits if the user cancels url input box', async () => {
+            test('return null if cluster url is not provided', async () => {
+                infoStub.resolves('Yes');
+                quickPickStub.onFirstCall().resolves(null);
+                const result = await Cluster.credentialsLogin();
+                expect(result).null;
+            });
+
+            test('return null if username is not provided', async () => {
+                infoStub.resolves('Yes');
+                quickPickStub.onSecondCall().resolves(null);
+                const result = await Cluster.credentialsLogin();
+                expect(result).null;
+            });
+
+            test("don't ask for save password if old and new password are same", async () => {
```

Comment:
don't => doesn't
for => to
password => passwords

`doesn't ask to save password if old and new passwords are the same`